packages/components/src/index.js

```diff
@@ -18,7 +18,7 @@ export { default as Draggable } from './draggable';
 export { default as DropZone } from './drop-zone';
 export { default as DropZoneProvider } from './drop-zone/provider';
 export { default as Dropdown } from './dropdown';
-export { default as DropdownMenu } from './dropdown-menu';
+export { default as DropdownMenu, DropdownMenuSeparator } from './dropdown-menu';
```

Comment:
I'm not sure if you had left it as a remaining task, but I'm expecting we don't want to be exposing this, based on prior remarks about semantics of menu groupings.

Reply:
Yes. I have must missed this one. I focused my efforts on bringing backward compatibility for More Menu.